### PR TITLE
Removed static protocolling for JS files.

### PR DESCRIPTION
**Pull request by yackermann**
I removed scripts static protocoling, because it was conflicting while using site on HTTPS. 
Replaced some links with HTTPS.

### CODE REVIEW

File: links.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ title: Links
...
 ========
 
-* [My current Github blog](http://joearms.github.io/")
-* [My old SICS Blog](http://www.sics.se/~joe)
-* [Armstrong on Software Blog](http://armstrongonsoftware.blogspot.se/)
-* [My old Bluetail blog](http://www.sics.se/~joe/bluetail/index.html)
-* [My old Ericsson Blog](http://www.sics.se/~joe/ericsson/index.html)
-* [My Professorial Blog](http://www.kth.se/profile/jlarm/)
+* [My current Github blog](https://joearms.github.io/")
+* [My old SICS Blog](https://www.sics.se/~joe)
```
Comment (herrniemand) [inline links.md]: Your old blog return 404. Maybe they need to be removed?*(

Comment (joearms) [reply links.md:14]: No sics should restore the files

Comment (herrjemand) [inline links.md]: @joearms Ok. It will fix that

Comment (yackermann) [reply links.md:14]: @joearms ok*)

**Pull request closed without merging**